### PR TITLE
added some comments, updated the name of login container, cleared the…

### PR DESCRIPTION
**Pull request by sadamexx**
… data from default data

# PR Description
​
Description of what is contained in pull request?  updated login container name, added comments to code, cleared the default data 
Description of why this change/addition is needed?  cleaning up for next team
Which user story or task is this PR tied to?  
How many files and/or lines were changed? 
​
​
## Type of Change
​
Types of Change:
- [] New feature
- [x] Refactoring / Bug fix
- [x] Documentation update
- [] Executive Change (technical decision change overwrite)
    - Reason for change (required):
​
​
## Testing
​
- [] Yes
- [] No
- [] Not needed
​
​
## Completeness
Is this feature complete
- [] Feature Complete
- [] Feature Incomplete
- [] User story/trello card linked here
​
# PR Requirements
- [] Clean and concise commenting
- [] Clean and comprehensive documentation included
- [] Conforms to engineering standards and checks
- [] There are no merge conflicts
- [] There are no warnings
- [] Corresponding documentation elsewhere has been updated to reflect these changes
​
​
## Requester Questions
​
```Add any questions, comments, or concerns you have as the person initiating the pull request for your reviewers```
​
​
## Other
[Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)  
[Template Guidelines](https://www.notion.so/Pull-Request-Template-f9264f79e1b649b9845961b5aba3eaff)
​

### CODE REVIEW

**Review by KristineYW — CHANGES_REQUESTED** [review]
Punctuation suggestions discussed in DMs. Thanks for fixing some of the placeholders that were left over from the last cohort!

**Comment by sadamexx** [comment]
Added periods per requested

File: src/components/pages/IntakePacket.jsx [hunk 1]
```diff
@@ -52,35 +52,35 @@ let defaultData = {
...
-      prior_location: '3211 East Ave',
-      current_location: 'nowhere',
-      num_times_homeless: 5,
-      total_len_homeless: '3 months',
-      homeless_start_date: '3/5/2018',
-      length_at_prior_location: '2 years',
-      length_at_current_location: '1 year',
+      prior_location: '',
+      current_location: '',
+      num_times_homeless: 0,
```
Comment (tajahouse) [inline src/components/pages/IntakePacket.jsx:79]: Nice! You even changed the number to 0 instead of leaving it blank! Great! Thank you!

**Review by tajahouse — APPROVED** [review]
Nice! It looks great Sarah!

**Pull request merged**